config-gen/src/main/java/org/wildfly/galleon/plugin/config/generator/ServerBridge.java

```diff
@@ -0,0 +1,274 @@
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.galleon.plugin.config.generator;
+
+import java.lang.reflect.Method;
+import org.wildfly.galleon.plugin.WfConstants;
+import org.wildfly.galleon.plugin.server.ConfigGeneratorException;
+
+public class ServerBridge {
+
+    private static final String ModelControllerClient = "org.jboss.as.controller.client.ModelControllerClient";
+    private static final String EmbeddedManagedProcess = "org.wildfly.core.embedded.EmbeddedManagedProcess";
+    private static final String EmbeddedProcessFactory = "org.wildfly.core.embedded.EmbeddedProcessFactory";
+    private static final String ModelNode = "org.jboss.dmr.ModelNode";
+    private static final String Operations = "org.jboss.as.controller.client.helpers.Operations";
+    private static final String ClientConstants = "org.jboss.as.controller.client.helpers.ClientConstants";
+
+    private static final String createHostController = "createHostController";
+    private static final String createStandaloneServer = "createStandaloneServer";
+    private static final String start = "start";
+    private static final String getModelControllerClient = "getModelControllerClient";
+    private static final String execute = "execute";
+    private static final String fromJSONString = "fromJSONString";
+    private static final String get = "get";
+    private static final String set = "set";
+    private static final String add = "add";
+    private static final String isSuccessfulOutcome = "isSuccessfulOutcome";
+    private static final String getFailureDescription = "getFailureDescription";
+    private static final String createCompositeOperation = "createCompositeOperation";
+    private static final String getProcessState = "getProcessState";
+    private static final String close = "close";
+    private static final String stop = "stop";
+    private static final String asString = "asString";
+    private static final String READ_ATTRIBUTE_OPERATION = "READ_ATTRIBUTE_OPERATION";
+    private static final String NAME = "NAME";
+    private static final String OP = "OP";
+    private static final String RESULT = "RESULT";
+
+    private static Class<?> EmbeddedManagedProcessClass;
+    private static Class<?> EmbeddedProcessFactoryClass;
+    private static Class<?> ModelControllerClientClass;
```

Comment:
Not critical, but these are never used outside the `initializeEmbedded()` method.